packages/xod-project/src/utils.js

```diff
@@ -31,6 +31,30 @@ export const formatString = R.curry((template, replacements) =>
 );
 
 /**
+ * Updates Error message of Left value by wrapping it with
+ * dead reference error message. Right value will be passed unchanged.
+ *
+ * Error message updated by mutation (to prevent creating new stack trace).
```

Comment:
I suggest moving this note closer to `// eslint-disable-next-line no-param-reassign`